fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -389,7 +393,7 @@ protected void resetTranslationChain() {
         if (translationChain == null) {
             translationChain = getTranslationChain();
             final List<Converter<String, String>> newChain =
-                    new ArrayList<>(singleton(new TransactionIdentifierConverter(session)));
+                    new ArrayList<>(singleton(new TransactionIdentifierConverter(transactionService, session)));
```

Comment:
No, the return of `singletonList` is immutable, don't worry about it. Like I said, I'll straighten this up later. Let's not tangle it in with the good stuff you are doing here.